### DISCUSSION TITLE
Indetectable property doesn't allow monsters to be undetected by humans

### DISCUSSION THREAD

**Discussion by TheNinjaScout** [FakeFishGames/Barotrauma#15018] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

A latched mantis which gains the "concealed" affliction can still be spotted and shot at by bots despite still being latched and concealed / camouflaged.

### Reproduction steps

1. Give a status effect to a monster to make it constantly concealed (or use a latched mantis)
2. Bring a crewmember near the character and note how they'll react to it despite the monster being concealed

### Bug prevalence

Just once

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by itchyOwl** [comment]
Fixed
